scripts/globals/job_utils/dragoon.lua

```diff
@@ -308,18 +309,18 @@ local function checkForRemovableEffectsOnSpiritLink(player, wyvern)
         -- Remove 2 erasable effects or effects that can be removed by -na
         local additionalRemovableEffects =
```

Comment:
This could be simplified a bit and use set{}

Reply:
never seen that syntax before. neat!